roles/openshift_metrics/tasks/install_metrics.yaml

```diff
@@ -1,6 +1,41 @@
 ---
 - include_tasks: pre_install.yaml
 
+- name: Get annotations from {{ openshift_metrics_old_project }}
+  oc_obj:
+    state: list
+    kind: project
+    name: "{{ openshift_metrics_old_project }}"
+    namespace: "{{ openshift_metrics_old_project }}"
+  register: openshift_metrics_infra
+
+- name:  Generate  template for project {{ openshift_metrics_project }} creation
+  template:
+    src: create_project.j2
+    dest: "{{ mktemp.stdout }}/templates/hawkular-metrics-projects.yaml"
+  vars:
+    project_name: "{{ openshift_metrics_project }}"
+    annotations: "{{ openshift_metrics_infra.results.results[0].metadata.annotations }}"
```

Comment:
One of the things I have been wondering is whether or not we should copy all of the annotations or just the 3 that we need. Here are the annotations from an lonline cluster:

```yaml
kind: Project
metadata:
  annotations:
    openshift.io/description: openshift-infra
    openshift.io/display-name: openshift-infra
    openshift.io/logging.ui.hostname: kibana-ops.c1b9.mycluster.openshiftapps.com
    openshift.io/node-selector: type=infra
    openshift.io/sa.initialized-roles: "true"
    openshift.io/sa.scc.mcs: s0:c3,c2
    openshift.io/sa.scc.supplemental-groups: 1000010000/10000
    openshift.io/sa.scc.uid-range: 1000010000/10000
```
I am not sure what the hostname annotation is before but I suspect that is something that does not get by the installer. I also wonder about the description and display-name annotations. If we do want those, I imagine they should refer to the new namespace. We can ask someone from the logging team about this.

Reply:
That is true, we only need 3 annotations according to this comment https://github.com/openshift/openshift-ansible/pull/8649#discussion_r193788678

If this PR works for you I can update the PR for 3.10 and we can ask.

I do not think is a big deal to filter this.

Reply:
Indeed I would prefer to set only those three. I feel more safe :)